docs/general/ad-filtering/create-own-filters.md

```diff
@@ -1341,345 +1499,194 @@ Use `@@` to negate `$removeheader`:
+>
+> In case if multiple `$replace` rules match a single request, we will apply each of them. **The order is defined alphabetically.**
+
+**Syntax**
+
+In general, `$replace` syntax is similar to replacement with regular expressions [in Perl](http://perldoc.perl.org/perlrequick.html#Search-and-replace).
 
 ```
-{
-    "elems": [
-        {
-            "a": {"b": {"c": 123}},
-            "k": "v"
-        },
-        {
-            "d": {"e": {"f": 123}},
-            "k1": "v1"
-        }
-    ]
-}
+replace = "/" regexp "/" replacement "/" modifiers
 ```
 
-</details>
+* **`regexp`** — a regular expression.
+* **`replacement`** — a string that will be used to replace the string corresponding to `regexp`.
+* **`modifiers`** — a regular expression flags. For example, `i` — insensitive search, or `s` — single-line mode.
 
-<details>
-<summary>Output</summary>
+In the `$replace` value, two characters must be escaped: comma `,` and dollar sign `$`. Use backslash `\` for it. For example, an escaped comma looks like this: `\,`.
+
+**Examples**
 
 ```
-{
-    "elems": [
-        {
-            "d": {"e": {"f": 123}},
-            "k1": "v1"
-        }
-    ]
-}
+||example.org^$replace=/(<VAST[\s\S]*?>)[\s\S]*<\/VAST>/\$1<\/VAST>/i
 ```
 
-</details>
+There are three parts in this rule:
+* `regexp` - `(<VAST(.|\s)*?>)(.|\s)*<\/VAST>`;
+* `replacement` — `\$1<\/VAST>` where `$` is escaped;
+* `modifiers` — `i` for insensitive search.
 
-* `||example.org^$jsonprune=\$.elems[?(key-eq "\$.a.b.c" "abc")]` removes all children of `elems` which have a property selectable by the JSONPath expression `$.a.b.c` with a value equal to `"abc"`.
+You can see how this rule works here:
+http://regexr.com/3cesk
 
-<details>
-<summary>Input</summary>
+**Multiple `$replace` rules**
 
-```
-{
-    "elems": [
-        {
-            "a": {"b": {"c": 123}},
-        },
-        {
-            "a": {"b": {"c": "abc"}}
-        }
-    ]
-}
-```
+1. `||example.org^$replace=/X/Y/`
+2. `||example.org^$replace=/Z/Y/`
+3. `@@||example.org/page/*$replace=/Z/Y/`
 
-</details>
+* Both rule 1 and 2 will be applied to all requests sent to `example.org`.
+* Rule 2 is disabled for requests matching `||example.org/page/`, **but rule 1 still works!**
 
-<details>
-<summary>Output</summary>
+**Disabling `$replace` rules**
 
-```
-{
-    "elems": [
-        {
-            "a": {"b": {"c": 123}}
-        }
-    ]
-}
-```
+* `@@||example.org^$replace` will disable all `$replace` rules matching `||example.org^`.
+* `@@||example.org^$document` or `@@||example.org^$content` will disable all `$replace` rules **originated from** pages of `example.org` **including the page itself**.
 
-</details>
+> **Compatibility with different versions of AdGuard**
+>
+> Rules with `$replace` modifier are supported by AdGuard for Windows, Mac, Android, and AdGuard Browser extension for Firefox. Such rules do not work in extensions for other browsers because they are unable to modify content on the network level.
```

Comment:
```suggestion
> Rules with `$replace` modifier are supported by AdGuard for Windows, Mac, and Android and AdGuard Browser Extension for Firefox. Such rules do not work in extensions for other browsers because they are unable to modify content on the network level.
```